packages/web-frontend/src/containers/MeasureLegend/common.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import styled from 'styled-components';
+
+export const LegendContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+`;
```

Comment:
These are entirely optional suggestions. 

Option 1
You could remove this re-usable component and use [material-ui box component](https://material-ui.com/components/box/) inline. eg 
```
<Box display="flex" alignItems="center" justifyContent="center" flexWrap="wrap">
`

Reply:
As a css newbie, I would much prefer to have it all spirited away in another file like `LegendContainer.js` than have to read it inline (but this is possibly just a personal thing which will go away with experience).